src/components/categories/categoriesSlice.js

```diff
@@ -0,0 +1,20 @@
+// categoriesSlice.js
+/* eslint-disable */
+
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: {
```

Comment:
- [ ] Kindly consider adding an initial state with an empty array, so the state can start as an empty array

Reply:
Implemented 👍